<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>R. Reucher</author>
 <class>Welcome</class>
 <widget class="QDialog" name="Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to QMC2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonOkay">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="comboBoxLanguage">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QFrame" name="frameLogo">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelLogoPixmap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="qmc2.qrc">:/data/img/qmc2_logo_big.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout2">
      <item row="5" column="2">
       <widget class="QToolButton" name="toolButtonBrowseSamplePath">
        <property name="toolTip">
         <string>Browse sample path</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditWorkingDirectory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Working directory that's used when the emulator is executed (if empty, QMC2's current working directory is used)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="toolButtonBrowseWorkingDirectory">
        <property name="toolTip">
         <string>Browse working directory</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelROMPath">
        <property name="text">
         <string>ROM path</string>
        </property>
        <property name="indent">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="labelSpacer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelExecutableFile">
        <property name="text">
         <string>Emulator executable</string>
        </property>
        <property name="indent">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="labelWelcome">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;h1&gt;Welcome to QMC2!&lt;/h1&gt;&lt;font size=&quot;+1&quot;&gt;&lt;p&gt;This appears to be your first start of QMC2 because no valid configuration was found. In order to use QMC2 as a front end for an emulator, you must specify the path to the emulator's executable file below.&lt;/p&gt;&lt;p&gt;The paths below the executable file are optional, but you should specify as many of them as you can right now to avoid problems or confusion later (of course, you can change the paths in the emulator's global configuration at any time later).&lt;/p&gt;&lt;p&gt;&lt;b&gt;It's strongly recommended that you specify the ROM path you are going to use at least!&lt;/b&gt;&lt;/p&gt;&lt;/font&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditHashPath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Path to hash files</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QToolButton" name="toolButtonBrowseHashPath">
        <property name="toolTip">
         <string>Browse hash path</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditROMPath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Path to ROM images</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="toolButtonBrowseExecutableFile">
        <property name="toolTip">
         <string>Browse emulator executable file</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelWorkingDirectory">
        <property name="text">
         <string>Working directory</string>
        </property>
        <property name="indent">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelHashPath">
        <property name="text">
         <string>Hash path</string>
        </property>
        <property name="indent">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="toolButtonBrowseROMPath">
        <property name="toolTip">
         <string>Browse ROM path</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelSamplePath">
        <property name="text">
         <string>Sample path</string>
        </property>
        <property name="indent">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditSamplePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Path to samples</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxExecutableFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Emulator executable file</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelLanguage">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <tabstops>
  <tabstop>comboBoxExecutableFile</tabstop>
  <tabstop>toolButtonBrowseExecutableFile</tabstop>
  <tabstop>lineEditWorkingDirectory</tabstop>
  <tabstop>toolButtonBrowseWorkingDirectory</tabstop>
  <tabstop>lineEditROMPath</tabstop>
  <tabstop>toolButtonBrowseROMPath</tabstop>
  <tabstop>lineEditSamplePath</tabstop>
  <tabstop>toolButtonBrowseSamplePath</tabstop>
  <tabstop>lineEditHashPath</tabstop>
  <tabstop>toolButtonBrowseHashPath</tabstop>
  <tabstop>pushButtonOkay</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>comboBoxLanguage</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>Welcome</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
